packages/material-ui/src/ImageListItemBar/ImageListItemBar.test.js

```diff
@@ -21,17 +20,107 @@ describe('<ImageListItemBar />', () => {
+      describe('position="top"', () => {
+        it('should render the positionTop class', () => {
+          const { container } = render(<ImageListItemBar title="text" position="top" />);
+
+          expect(container.querySelector('div')).to.have.class(classes.root);
+          expect(container.querySelector('div')).to.have.class(classes.positionTop);
+        });
+      });
+
+      describe('position="below"', () => {
+        it('should render the positionBelow class', () => {
+          const { container } = render(<ImageListItemBar title="text" position="below" />);
+
+          expect(container.querySelector('div')).to.have.class(classes.root);
+          expect(container.querySelector('div')).to.have.class(classes.positionBelow);
+        });
+
+        it('should render the positionBelowSubtitle class', () => {
+          const { container } = render(
+            <ImageListItemBar title="text" subtitle="subtitle" position="below" />,
+          );
+
+          expect(container.querySelector('div')).to.have.class(classes.root);
+          expect(container.querySelector('div')).to.have.class(classes.positionBelowSubtitle);
+        });
+
+        it('should render a child div with the titleWrapBelow class', () => {
+          const { container } = render(<ImageListItemBar title="text" position="below" />);
+
+          expect(container.firstChild.querySelector('div')).to.have.class(classes.titleWrap);
+          expect(container.firstChild.querySelector('div')).to.have.class(classes.titleWrapBelow);
+        });
+      });
+    });
+
+    describe('prop: actionPosition', () => {
+      it('should render a child div with the titleWrapActionPosLeft class', () => {
+        const { container } = render(
+          <ImageListItemBar title="text" actionIcon={<div />} actionPosition="left" />,
+        );
+
+        expect(container.firstChild.querySelector('div')).to.have.class(classes.titleWrap);
+        expect(container.firstChild.querySelector('div')).to.have.class(
+          classes.titleWrapActionPosLeft,
+        );
+      });
+    });
+  });
+
+  describe('classes', () => {
+    it('should render with the root and positionBottom classes by default', () => {
+      const { container } = render(<ImageListItemBar title="text" />);
+
+      expect(container.querySelector('div')).to.have.class(classes.root);
+      expect(container.querySelector('div')).to.have.class(classes.positionBottom);
+    });
+
+    it('should render a child div with the titleWrap class by default', () => {
+      const { container } = render(<ImageListItemBar title="text" />);
+
+      expect(container.firstChild.querySelector('div')).to.have.class(classes.titleWrap);
+    });
+
+    it('should render a child div with the titleWrapActionPosRight class', () => {
+      const { container } = render(<ImageListItemBar title="text" actionIcon={<div />} />);
+
+      expect(container.firstChild.querySelector('div')).to.have.class(classes.titleWrap);
```

Comment:
Not sure about selector like `container.firstChild.querySelector('div')`

Reply:
What would you suggest?

Reply:
Do we need these tests in the first place? What type of bugs would they help prevent?